noggin/controller/password.py

```diff
@@ -138,7 +138,10 @@ def forgot_password_ask():
                 raise FormError(
                     "username", _("User %(username)s does not exist", username=username)
                 )
-            token = PasswordResetToken.from_user(user).as_string()
+            token = make_token(
+                {"sub": user.username, "lpc": user.last_password_change},
+                audience="pwdrst",
```

Comment:
Am I right assuming that the valid values of `audience` are defined in Basset? I had to think hard what `pwdrst` means (and I'm at an advantage, I see what this was before :wink:).

Reply:
No, audience values are internal, it just needs to be the same on token creation and token parsing side, to avoid token re-use. We can use whatever we want but using too long a string has disadvantages because it makes the token longer, and it ends up in the URL so there's a size limit. I could use an enum if you think it makes it clearer.

Reply:
Maybe it's just my brain taking the wrong turn when coming across anything `pwd...`, maybe we could use `passreset` (which is only one character more)?

Reply:
I switched to an enum, it should make things clearer :-)